src/Arcus.Templates.AzureFunctions.Http/HttpBasedAzureFunction.cs

```diff
@@ -1,209 +1,191 @@
-                ContentTypes = { "application/json" }
-            };
-        }
-
-        /// <summary>
-        /// Creates an <see cref="IActionResult"/> instance for a '415 Unsupported Media Type' failure with an accompanied <paramref name="errorMessage"/>.
-        /// </summary>
-        /// <param name="errorMessage">The error message to describe the failure.</param>
-        /// <exception cref="ArgumentException">Throw when the <paramref name="errorMessage"/> is blank.</exception>
-        protected IActionResult UnsupportedMediaType(string errorMessage)
-        {
-            Guard.NotNullOrWhitespace(errorMessage, nameof(errorMessage), "Requires a non-blank error message to accompany the 415 Unsupported Media Type failure");
-            return new ObjectResult(errorMessage)
-            {
-                StatusCode = StatusCodes.Status415UnsupportedMediaType
-            };
-        }
-
-        /// <summary>
-        /// Creates an <see cref="IActionResult"/> instance for a '400 Bad Request' failure with an accompanied <paramref name="error"/> model to describe the failure.
-        /// </summary>
-        /// <param name="error">The error model to describe the failure.</param>
-        /// <exception cref="ArgumentNullException">Thrown when the <paramref name="error"/> is <c>null</c>.</exception>
-        protected IActionResult BadRequest(object error)
-        {
-            Guard.NotNull(error, nameof(error), "Requires an error model to describe the 400 Bad Request failure");
-            return new BadRequestObjectResult(error);
-        }
-
-        /// <summary>
-        /// Creates an <see cref="IActionResult"/> instance for a '500 Internal Server Error' failure with an accompanied <paramref name="errorMessage"/>.
-        /// </summary>
-        /// <param name="errorMessage">The error message to describe the failure.</param>
-        /// <exception cref="ArgumentException">Thrown when the <paramref name="errorMessage"/> is blank.</exception>
-        protected IActionResult InternalServerError(string errorMessage)
-        {
-            Guard.NotNullOrWhitespace(errorMessage, nameof(errorMessage), "Requires an non-blank error message to accompany the 500 Internal Server Error failure");
-            return new ObjectResult(errorMessage)
-            {
-                StatusCode = StatusCodes.Status500InternalServerError,
-                ContentTypes = {"text/plain"}
-            };
-        }
-    }
-}
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+using Arcus.WebApi.Logging.Correlation;
+using GuardNet;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Formatters;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Net.Http.Headers;
+

```

Comment:
Is it just me, or is this diff ui not working as expected ?

Reply:
Yeah, something about new line chars that are probably not right. I fixed it locally, so it won't happen again.